lib/controllers/flow/account_linking_flow_controller.dart

```diff
@@ -87,6 +87,9 @@ class AccountLinkingFlowController extends GetxController {
       case ConsentStatus.revoked:
         _stopListening();
         break;
+      default:
+        // we don't care about the other statuses
```

Comment:
will we never care? 
also if there's a status that's not in the enum shouldn't we be throwing/loggin an error?